src/index.js

```diff
@@ -11,4 +11,38 @@ function scoreElement(name, score) {
   tableRow.appendChild(tableData);
   tableBody.appendChild(tableRow);
 }
-scores.forEach((score) => scoreElement(score.user, score.score));
+async function getScores() {
+  // Get scores data
```

Comment:
- [ ] Kindly make use of arrow functions instead of the function keyword to follow ES6 best practices.